controller/work_item_link_type_combination.go

```diff
@@ -0,0 +1,187 @@
+package controller
+
+import (
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/application"
+	"github.com/almighty/almighty-core/jsonapi"
+	"github.com/almighty/almighty-core/rest"
+	"github.com/almighty/almighty-core/workitem/link"
+	"github.com/goadesign/goa"
+	errs "github.com/pkg/errors"
+)
+
+// WorkItemLinkTypeCombinationController implements the work_item_link_type_combination resource.
+type WorkItemLinkTypeCombinationController struct {
+	*goa.Controller
+	db     application.DB
+	config WorkItemLinkTypeCombinationControllerConfiguration
+}
+
+// WorkItemLinkTypeCombinationControllerConfiguration the configuration for the WorkItemLinkTypeController
+type WorkItemLinkTypeCombinationControllerConfiguration interface {
+	GetCacheControlWorkItemLinkTypeCombinations() string
+}
+
+// NewWorkItemLinkTypeCombinationController creates a work_item_link_type_combination controller.
+func NewWorkItemLinkTypeCombinationController(service *goa.Service, db application.DB, config WorkItemLinkTypeCombinationControllerConfiguration) *WorkItemLinkTypeCombinationController {
+	if db == nil {
+		panic("db must not be nil")
```

Comment:
Okay, fixed in 5b51c4b8e441e500ea11980395916af4c66edcce. But notice that I wasn't the only one testing for `db == nil` ;)